iceoryx_examples/callbacks/ice_callbacks_subscriber.cpp

```diff
@@ -100,14 +101,26 @@ int main()
     });
 
     // attach everything to the listener, from here on the callbacks are called when the corresponding event is occuring
-    listener.attachEvent(heartbeat, heartbeatCallback);
-    listener.attachEvent(subscriberLeft, iox::popo::SubscriberEvent::HAS_DATA, onSampleReceivedCallback);
+    listener.attachEvent(heartbeat, heartbeatCallback).or_else([](auto) {
+        std::cerr << "unable to attach heartbeat event" << std::endl;
```

Comment:
not sure if it would be better to use the logger instead

Reply:
In the examples I would keep it as simple as possible. Otherwise we also have to create an introduction example for our logger or we have to explain it here somehow but `std::cerr` is understood by everyone.